config/events-web.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Csrf\CsrfToken;
+
+return [
+    \Yiisoft\Yii\Web\Event\BeforeRequest::class => [
+        fn (object $event, ContainerInterface $container) => CsrfToken::initialize(),
```

Comment:
```suggestion
        fn (\Yiisoft\Yii\Web\Event\BeforeRequest::class $event, ContainerInterface $container) => CsrfToken::initialize(),
```

Reply:
You don't use `ContainerInterface $container` in this case, remove it